.github/CODEOWNERS

```diff
@@ -38,6 +38,7 @@ README.md @ComposableFi/technical-writing-reviewers @ComposableFi/core @Composab
 ## Frontend and Blockchain Integration
 
 frontend/ @ComposableFi/blockchain-integrations @ComposableFi/parachain-finance
+subsquid/ @ComposableFi/blockchain-integrations
```

Comment:
There is another line in this file for subsquid, you can modify it and add the new group

Reply:
You are right, I didn't see it